as_perimenoun_oi_gynaikes.ass

```diff
@@ -25,15 +25,15 @@ Style: Default,Arial,22,&H00FFFFFF,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,
 
 [Events]
 Format: Layer, Start, End, Style, Name, MarginL, MarginR, MarginV, Effect, Text
-Dialogue: 0,0:00:03.74,0:00:08.74,Default,,0,0,0,,"Foundries Makedonia"\NPanos Karagiannakidis\NMichail Arbazoglou\NThessaloniki
+Dialogue: 0,0:00:03.74,0:00:08.74,Default,,0,0,0,,"Foundries Macedonia"\NPanos Karagiannakidis\NMichail Arbazoglou\NThessaloniki
 Dialogue: 0,0:00:32.12,0:00:34.24,Default,,0,0,0,,This is where you'll lay your dust!
 Dialogue: 0,0:00:34.40,0:00:37.44,Default,,0,0,0,,Here! You punks! You gypsy-dog iron smiths!
 Dialogue: 0,0:00:37.44,0:00:42.32,Default,,0,0,0,,Animal control will get to you eventually, you clowns!
 Dialogue: 0,0:00:42.32,0:00:49.40,Default,,0,0,0,,Swedish girls, aged fifteen to eighteen. \NFrom their mom's breast straight to consumption.
-Dialogue: 0,0:00:49.40,0:00:52.08,Default,,0,0,0,,Marika will straighten you out. Start with the washing.
+Dialogue: 0,0:00:49.40,0:00:52.08,Default,,0,0,0,,Marika will force you to do the laundry, you clown!
 Dialogue: 0,0:00:52.08,0:00:55.04,Default,,0,0,0,,Speedboat-rides under the moonlight.
-Dialogue: 0,0:00:59.48,0:01:02.12,Default,,0,0,0,,Boo, you herdsman!
-Dialogue: 0,0:01:02.12,0:01:06.04,Default,,0,0,0,,Here we'll erect your grave! Here we'll erect your grave!
+Dialogue: 0,0:00:59.48,0:01:02.12,Default,,0,0,0,,Boo, you undemanding fucker!

```

Comment:
"Herdsman" is more appropriate here. https://www.urbandictionary.com/define.php?term=Herdsman